frank/plot.py

```diff
@@ -31,308 +31,205 @@
 from frank.utilities import sweep_profile
 
 
-def plot_brightness_profile(fit_r, fit_i, ax, yscale='linear', c='r', ls='-',
-                            ylolim=None, alpha=1., label='Frank'
-                            ):
-    # TODO: take in (and move all these to) kwargs for standard plot params. do throughout script. then clean up docstrings
+def plot_deprojection_uv(u, v, up, vp, ax):
+    """
+    Overplot projected and deprojected (u, v) coordinates
+
+    Parameters
+    ----------
+    u, v : array
+        Projected (u, v) coordinates
+    up, vp : array
+        Deprojected (u, v) coordinates
+    ax : Matplotlib `~.axes.Axes` class
+        Axis on which to plot
+    """
+
+    ax.plot(u, v, '+', c='#1EC8FE', label='Projected')
+    ax.plot(up, vp, 'x', c='#D14768', label='Deprojected')
+
+    ax.legend(loc='best')
```

Comment:
88f6d8a and a2031ac combine `plot_deprojection_vis` and `plot_deprojection_uv` into `plot_deprojection_effect`.

I'd like to keep it so that `make_deprojection_fig` doesn't plot anything explicitly, just calls the functions in `plot` to do this (as is the case for the rest of the functions in `make_figs`).

Reply:
ok